client/galaxy/scripts/apps/analysis.js

```diff
@@ -83,6 +84,7 @@ window.app = function app( options, bootstrapped ){
             '(/)user(/)(:form_id)' : 'show_user_form',
             '(/)workflow(/)' : 'show_workflows',
             '(/)pages(/)(:action_id)' : 'show_pages',
+            '(/)histories(/)(:action_id)' : 'show_histories',
```

Comment:
Sorry, should have commented earlier. This is a preparation for `published_lists`. However you are right that in the earlier PR for datasets it is not necessary, since there is no published datasets grid. I will remove it from the other one.